identity-service-logs/utils.js

```diff
@@ -214,6 +214,10 @@ async function getSelfUser() {
     });
 
     const self_user = await res.json();
+    if (self_user?.statusCode !== 200) {
+      alert('You are not loggedin. Please login!');
+      window.location.href = '/index.html';
+    }
     return self_user;
```

Comment:
ncorrect Error Handling:
The catch block returns an empty object ({}) when an error occurs. This might lead to unexpected behavior in the calling code. It's generally recommended to rethrow the error or return a specific error object for proper error handling.